### ISSUE TITLE
[TF2][Feature Request] Votes should complete as soon as the outcome is guaranteed

### ISSUE THREAD

**Issue by okamiterasu** [ValveSoftware/Source-1-Games#4002]
It seems silly that a vote that has already reached a majority should have to wait for the remainder of the vote timer to run out before completing. It unnecessarily slows things down and, in the case of vote kicks, allows bad actors that much more time to cause havoc.

**Comment by AzureAzel** [comment]
Agreed. Additionally, I'd also prefer it if the cooldown for votekicks was lessened a bit.

**Comment by TF2CutContentWiki** [comment]
Third'd. Would help with bots.

**Comment by jh34ghu43gu** [comment]
Of note is I have observed a weighted vote power within casual, not sure if it's related to premium accounts or how many times someone has been recently vote kicked, hours, etc.. It is also possible the unvoted people's "absentee vote" is used in these weighted calculations on if a vote passes or not despite and making them instantly pass once the flat >50% of yes votes is reached may break this. 

For clarification on weighted votes, I have survived multiple vote kicks of >50% yes in majority bot lobbies (ie 8v3 votes) which would suggest some sort of behind-the-scenes weighting going on.

**Comment by MeatwadMeatwad** [comment]
Necroing because this is something that should be added. I encounter this problem a lot where most of the time a bot is active and killing the enemy team is when we are waiting for the vote to end, even if all the necessary votes are there.
